packages/framework-integration-tests/src/config/config.ts

```diff
@@ -28,4 +29,5 @@ Booster.configure('production', (config: BoosterConfig): void => {
 
   config.appName = 'my-store-' + appNameSuffix
   config.provider = AWS.Provider()
+  config.staticResourceFolders = ['public']
```

Comment:
The user configures an array of folders that will be zipped with the lambda code. Later they will be able to read it inside command and event handlers